usr/klibc/arch/x86_64/setjmp.S

```diff
@@ -17,6 +17,10 @@
 #
 
 	.text
+#include <asm/nospec-branch.h>
+
+#define RET jmp __x86_return_thunk
```

Comment:
I don't think it's necessary.. if I recall correctly, it's defined in the header above